src/app/(cities)/wroclaw/page.mdx

```diff
@@ -37,4 +42,49 @@ export const metadata = {
 	},
 };
 
-# Wrocław
\ No newline at end of file
+<CityBanner city='Wrocław' background='wroclaw.jpeg' />
+
+<LocalGroups
+	localGroups={['https://www.meetup.com/meet-js-wroclaw/', 'https://www.facebook.com/groups/326774397348491', 'https://lu.ma/user/usr-eNzoaGXYjizOBQI']} />
+
+<FAQ questions={[{ id: '1', question: 'How to become Wrocław meetup sponsor?', answer: 'contact' }, {
```

Comment:
Should we save it in a different file? I assume we won't create any back-office soon or anytime, so it's okay, but I would put it into a separate file.

Reply:
Every city in different file?

Reply:
data for FAQ in separate file

Reply:
Overall we can merge it. It's not a blocker, just a suggestion.